examples/reference.yml

```diff
@@ -134,6 +134,16 @@ ldap_auth:
   # User query settings. ${account} is expanded from auth request 
   base: o=example.com
   filter: (&(uid=${account})(objectClass=person))
+  # Labels can be mapped from LDAP attributes
+  labels:
+    # Add the user's title to a label called title
+    title:
+      attribute: title
+    # Add the user's memberOf values to a label called groups
+    groups:
+      attribute: memberOf
+      # Special handling to simplify the values to just the common name
+      parse_cn: true
```

Comment:
Good, I would also suggest to provide an example at acl section of how to use these attributes

Reply:
For those who are not very familiar with LDAP, it would be nice to provide example when `parse_cn` should be `true` and when `false` (false is default behavior, right)?

Reply:
It is a special case _only_ for `cn` fields, it isn't very reusable

Reply:
That's just to get `foo` from `cn=foo,ou=bar,dc=bar`, right?